Error on RegisterExtension

During import I am seeing this: 
```
[ERROR] AttributeError: type object 'FeedHeader' has no attribute 'RegisterExtension'
Traceback (most recent call last):
  File "/var/lang/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/task/merge.py", line 20, in <module>
    from parsers.mta_trips import gtfs_files, mta_trips
  File "/var/task/parsers/mta_trips.py", line 4, in <module>
    from nyct_gtfs import NYCTFeed
  File "/var/task/nyct_gtfs/__init__.py", line 1, in <module>
    from nyct_gtfs.feed import NYCTFeed
  File "/var/task/nyct_gtfs/feed.py", line 7, in <module>
    from nyct_gtfs.compiled_gtfs import nyct_subway_pb2, gtfs_realtime_pb2
  File "/var/task/nyct_gtfs/compiled_gtfs/nyct_subway_pb2.py", line 62, in <module>
    gtfs__realtime__pb2.FeedHeader.RegisterExtension(nyct_feed_header)
```

--- a/setup.py
+++ b/setup.py
@@ -28,5 +28,5 @@ setup(
     package_data={
         "nyct_gtfs": ["gtfs_static/*.txt"]
     },
-    install_requires=["requests", "protobuf", "httpx"]
+    install_requires=["requests", "protobuf==4.25.3", "httpx"]
 )

